challenges/desafio11.js

```diff
@@ -0,0 +1 @@
+db.voos.find({ "aeroportoDestino.pais": { $ne: "ESTADOS UNIDOS" } }).count();
```

Comment:
Boaaa, uma outra forma de fazer seria colocando a condição deretamente no count ai vc não precisaria do find.
```suggestion
db.voos.count({ "aeroportoDestino.pais": { $ne: "ESTADOS UNIDOS" } });
```